resources/lib/NineAnimeBrowser.py

```diff
@@ -116,9 +129,14 @@ def get_genre(self, name, page=1):
         return self._process_anime_view(url, data, "genre/%s/%%d" % name, page)
 
     def get_anime_episodes(self, anime_url):
+        settings = xbmcaddon.Addon(id='plugin.video.9anime')
```

Comment:
you are getting the value exactly as supposed.
next, call _BROWSER.get_anime_episodes and save it into a list on this function.
handle the list depends on reverseorder value, and only finally return the control.draw_items